llama_toolchain/agentic_system/meta_reference/system_prompt.py

```diff
@@ -5,21 +5,27 @@
 # the root directory of this source tree.
 
 import json
+import textwrap
 from datetime import datetime
 from typing import List
 
+from llama_toolchain.agentic_system.api.datatypes import ToolPromptFormat
+
 from llama_toolchain.inference.api import (
     BuiltinTool,
     Message,
     SystemMessage,
     ToolDefinition,
+    UserMessage,
 )
 
 from .tools.builtin import SingleMessageBuiltinTool
 
 
 def get_agentic_prefix_messages(
```

Comment:
so this code is duplicated in a sense -- shouldn't we take this opportunity to point this code (or relocate this formatting code) to `llama-models`?

Reply:
not clear if this should be in llama-models as its specific to agentic / tools but either ways its a bit out of scope for this PR. Going to merge this and open another one for this if needed.